static/app/views/performance/queues/messageSamplesPanel.tsx

```diff
@@ -23,22 +24,26 @@ import {MetricReadout} from 'sentry/views/performance/metricReadout';
 import * as ModuleLayout from 'sentry/views/performance/moduleLayout';
 import {MessageSpanSamplesTable} from 'sentry/views/performance/queues/messageSpanSamplesTable';
 import {useQueuesMetricsQuery} from 'sentry/views/performance/queues/queries/useQueuesMetricsQuery';
+import {
+  CONSUMER_QUERY_FILTER,
+  MessageActorType,
+  PRODUCER_QUERY_FILTER,
+} from 'sentry/views/performance/queues/settings';
+import {Subtitle} from 'sentry/views/profiling/landing/styles';
 import {computeAxisMax} from 'sentry/views/starfish/components/chart';
 import DetailPanel from 'sentry/views/starfish/components/detailPanel';
 import {useSpanMetricsSeries} from 'sentry/views/starfish/queries/useDiscoverSeries';
-import {SpanIndexedField} from 'sentry/views/starfish/types';
+import {SpanIndexedField, type SpanMetricsResponse} from 'sentry/views/starfish/types';
 import {useSampleScatterPlotSeries} from 'sentry/views/starfish/views/spanSummaryPage/sampleList/durationChart/useSampleScatterPlotSeries';
 
-// We're defining our own query filter here, apart from settings.ts because the spans endpoint doesn't accept IN operations
-const DEFAULT_QUERY_FILTER = 'span.op:queue.process OR span.op:queue.publish';
-
-export function MessageConsumerSamplesPanel() {
+export function MessageSamplesPanel() {
   const router = useRouter();
   const query = useLocationQuery({
     fields: {
       project: decodeScalar,
       destination: decodeScalar,
       transaction: decodeScalar,
+      op: decodeScalar,
```

Comment:
`op` is perfectly fine, but `span.op` is a little nicer (even though it's annoying to type) in case people need to search for it, and it's more consistent with how things are filtered everywhere else

Reply:
Updated!